src/accounts/providers/JsonRPCWallet.ts

```diff
@@ -10,10 +9,12 @@ export enum RpcChainType {
     POLYGON,
     BSC,
     AVAX,
+    AVAX_TESTNET,
 }
 
 export type RpcType = {
-    chainId: string;
+    chainIdHex: string;
+    chainIdDec: number;
```

Comment:
Maybe will be better to just save it in one format as before and convert it to desired format when is needed or maybe convert it on the initialization.

Reply:
Indeed, this can cause trouble otherwise